app/javascript/src/components/Dashboard/Contacts/SideMenu.jsx

```diff
@@ -8,7 +8,7 @@ const SideMenu = ({ showContactMenu }) => {
   const [isSearchCollapsed, setIsSearchCollapsed] = useState(true);
 
   return (
-    <div className="flex">
+    <div>
```

Comment:
Why is this div required here?

Reply:
I thought of putting div as a single root element. 
But yeah after removing div, I got to know  ```MenuBar``` itself is also a root element.
I have rectified it.
Thank you.

Reply:
@aryanandarnav13 https://www.loom.com/share/5285e065e19d4224a9819822ab571eed

Reply:
Ok, Yedhin.

Reply:
Sorry, @yedhink for the confusion. 
I have explained what I understood.

https://www.loom.com/share/5c6f95a396af4d41be4d49d05d588143

Reply:
Got it. Thanks for the video explanation. Appreciate it.